build.sbt

```diff
@@ -43,7 +43,12 @@ lazy val root = project
     // publish settings
     releaseCrossBuild := true,
     skip in publish := true,
-    publishTo := sonatypePublishTo.value,
+    publishTo := Some(
+      if (isSnapshot.value)
+        Opts.resolver.sonatypeSnapshots
+      else
+        Opts.resolver.sonatypeStaging
+    ),
```

Comment:
What do these changes do?

Reply:
@nicl can you help with this?

Reply:
If it's a SNAPSHOT (pre) release, then publish to sonatype's snapshots repository. 

https://help.sonatype.com/repomanager3/formats/maven-repositories

Though, I'm not sure if we actually use Snapshots, so you could just drop this change.

Reply:
The last snapshot we had for this is V0.0.1-SNAPSHOT. @akash1810 are we likely to use snapshot versions for this repository in the future? If not let me know and I will remove the above code.

Reply:
Thanks for the explanation.

> Though, I'm not sure if we actually use Snapshots

Same. I think we tend to `publishLocal` or fix forward.

Looking at the revision history of this repository, I wouldn't anticipate much activity. That is, I don't think use of the snapshot repository is needed.

Reply:
Super -  I'll remove this bit. Thanks all.